src/OrbitQt/orbitmainwindow.ui

```diff
@@ -487,17 +496,20 @@ QPushButton:disabled {
    <property name="toolTip">
     <string>Toggle Capture</string>
```

Comment:
As an idea: Maybe we should also indicate the shortcuts in these tool tips here. 
"Toggle Capture (Ctrl+ X)" 
(Same for help)

Reply:
Good idea, I thought QT would display them automatically, but apparently not. I've added them, thanks.

Reply:
They are added in the file menu automatically, but apparently not in the tooltips.